repondeur/zam_repondeur/fetch/an/dossiers/dossiers_legislatifs.py

```diff
@@ -108,9 +111,16 @@ def gen_lectures(acte: dict, textes: Dict[str, Texte]) -> Iterator[Lecture]:
         assert result.texte is not None
         texte = textes[result.texte]
 
-        yield Lecture(  # type: ignore
-            chambre=chambre, titre=titre, texte=texte, organe=result.organe
-        )
+        parties: List[Optional[int]] = [1, 2] if is_plf else [None]
+
+        for partie in parties:
+            yield Lecture(  # type: ignore
```

Comment:
I wonder if we should uniformize creations with the `model.create()` method. 🤔

Reply:
Here it’s not an SQLAlchemy model, it’s a dataclass

Reply:
Indeed 🤦‍♂️

Reply:
I got confused too when looking bad at that code